yogstation/code/game/objects/items/devices/scanners.dm

```diff
@@ -41,7 +41,8 @@ obj/item/multitool/tricorder/suicide_act(mob/living/carbon/user)
 //Tricorder differentiates from slimes and nonslimes
 /obj/item/multitool/tricorder/attack(mob/living/M, mob/living/user, obj/item/I)
 	add_fingerprint(user)
-	atmosanalyzer_scan()
+	var/turf/T = get_turf(M)
```

Comment:
Grab the turf of the target I item
Choose another name than T as it's already in use